extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -400,16 +400,23 @@ export const AzureProvisionDialog: React.FC = () => {
         );
       }
     } else {
+      // TODO: handle when existing profile is being edited
```

Comment:
is @GeoffCoxMSFT aware of this?

Reply:
I don't know. I plan on creating an issue to track.